### DISCUSSION TITLE
Install Shelly firmware on Tasmotized "Shelly 1"

### DISCUSSION THREAD

**Discussion by DJF3** [arendst/Tasmota#19552] [Q&A]
Team,
I have a bunch of Shelly 1's that I will give to someone who can really use them. However, in that scenario I want them to run the default Shelly firmware. Right now they are running Tasmota 12.02. 

What is the procedure to "downgrade" :-) them to Shelly FW?

**Comment by sfromis** [comment]
https://github.com/yaourdt/tasmota-to-mgos exists. Not updated recently.

**Comment by Jason2866** [comment]
Tasmota crew provides no tool or procedure to reinstall Shelly firmware.
Maybe Shelly recovery https://www.shelly-support.eu/filebase/index.php?filebase/49-auslieferungs-fw/
